docs/source/conf.py

```diff
@@ -13,7 +13,15 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath("../../src"))
+# -- Options specific to readthedocs -----------------------------------------
+
+on_readthedocs = os.environ.get("READTHEDOCS") == "True"
+if not on_readthedocs:
+    # If not invoked from the `readthedocs` build environment, use the source
+    # files instead of assuming the package is installed, and mock `rpu_base`.
+    import sys
+    sys.path.insert(0, os.path.abspath("../../src"))
+    autodoc_mock_imports = ["aihwkit.simulator.rpu_base"]
```

Comment:
Small re-hash of some bits that were removed at #585 . In order to correctly produce the `.html` files for `rpu_base`, `autosummary` needs to work on the installed package (as, if working on just the `.py` files without installing, it will not be able to resolve the reference). If not inside the readthedocs environment, I aimed at preserving the previous behaviour.